node/src/chain_spec.rs

```diff
@@ -57,14 +57,14 @@ pub fn create_pendulum_multisig_account(id: &str) -> AccountId {
 /// The extensions for the [`ChainSpec`].
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize, ChainSpecGroup, ChainSpecExtension)]
 #[serde(deny_unknown_fields)]
-pub struct Extensions {
+pub struct ParachainExtensions {
```

Comment:
I renamed this to distinguish from a standalone chain spec, which does not have this chain spec extension.